app/views/pages/get-started/working-with-the-code-and-files.liquid

```diff
@@ -0,0 +1,178 @@
+
+
+## Upload code changes automatically when you save the file
+
+On a daily basis, you want the instance to immediately reflect the changes you made in the code with a minimum effort on your side.
+
+This process we call **syncing** and it’s handled by a pos-cli command ran from the root project’s directory:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync &lt;environment&gt;
+</code></pre>
+
+When executing this command you will learn that it does not end by itself. This is a command that you want to run and leave opened while you make code changes.
+
+What `pos-cli sync` is doing is basically observing your codebase, so when you save changes in a file – it automatically uploads it to your instance. Just the file that changed so it’s quick and efficient!
+
+
+### Syncing in action
+
+For our example we can try another change in the code, but before we’ll do it – let’s start the sync using the following command:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync dev
+</code></pre>
+
+A confirmation message saying that the synchronization has been started should appear, so we can find another place in the code to test how this works. Please find the `Hello world` title in `index.html.liquid` of the demo page and change it to something else, like `Hello beautiful world`.
+
+After the change, just save the file.
+
+In the command line you should see a confirmation message saying that the edited page was synced. Refreshing the page in the browser should confirm that everything worked and the new text content is in place.
+
+Syncing handles not only code changes, but also all the other assets. Images, JavaScript files or even GraphQL queries – any change in your project directories will be automatically reflected on your instance.
+
+
+## Basic overlook on the directory structure
+
+The `pos-cli init` command will create all of the required folders used by platfromOS. The very basic directories you need to be aware of are those used to store files, web pages and handling data.
+
+We have a whole separate article [describing each directory](/developer-guide/platformos-workflow/codebase) function in detail if you want to jump directly into the technical aspect. Those four mentioned below are enough to start though.
+
+### Assets
+
+Every **static**, publicly available, **asset** is stored in the app/assets folder. Images, JavaScript, fonts, CSS files or any downloadable documents should go here. We recommend a standarized set of subfolders for each type, so:
+
+- app/assets/images for images
+- app/assets/scripts for JavaScript files
+- app/assets/styles for the page CSS
+- app/assets/fonts for the font files
```

Comment:
Still being discussed

Reply:
How about a wording closer to "You can organize your assets in any way you want, as long as they are in the app/assets. For example, you might want to use the following structure to separate various asset types from each other:" ?